prov/efa/configure.m4

```diff
@@ -113,6 +115,44 @@ AC_DEFUN([FI_EFA_CONFIGURE],[
 	AC_SUBST(efa_CPPFLAGS)
 	AC_SUBST(efa_LDFLAGS)
 	AC_SUBST(efa_LIBS)
+
+	AC_ARG_WITH([cmocka],
```

Comment:
It seems you are using `--with-cmocka` to indicate that efa unit test will be built. That is confusing. The command line argument should be `--with-efa-unit-test`, user can point cmocka directory to that. e.g. 

`--with-efa-unit-test=/path/to/cmocka`

Reply:
switched to `--enable-efa-unit-test=/path/to/cmocka`